data/2mass_j11074656-7615174.json

```diff
@@ -7,7 +7,7 @@
             "epoch": 2000.0,
             "equinox": "J2000",
             "shortname": null,
-            "reference": "Góme01",
+            "reference": "G\u00f3me01",
```

Comment:
Oops. This went the wrong way!

Reply:
Yea, it's weird but a consequence. It is fixed in the second PR, though.